scripts/ci/in_container/_in_container_utils.sh

```diff
@@ -195,4 +195,5 @@ EOF
 
 function stop_output_heartbeat() {
     kill "${HEARTBEAT_PID}"
+    wait "${HEARTBEAT_PID}" || true 2> /dev/null
```

Comment:
What do you think to start using `jobs/fg/bg/kill %1` here? Running scripts this way is a bit hacky to me.

Reply:
Kill %1 is the same as PID but it is really intended for interactive use (when you see the output of `jobs` command). You never know which job you are killing this way.  If you have some other background jobs started before your  "start_output_heartbeat" then `kill %1` will kill the other job, not the heartbeat. Storing PID of the started process and using it explicitly is the only sure way I know to kill exactly the process I intend to.